PurchasesCoreSwift/Attribution/AdClientProxy.swift

```diff
@@ -38,18 +37,15 @@ class FakeAdClient: NSObject {
 
 }
 
-// TODO (post-migration): Make all the things internal again.
-@objc(RCAdClientProxy)
-public class AdClientProxy: NSObject {
+class AdClientProxy: NSObject {
 
     private static let className = "ADClient"
 
     static var adClientClass: AnyClass? {
         NSClassFromString(Self.className)
     }
 
-    @objc(requestAttributionDetailsWithBlock:)
-    public func requestAttributionDetails(_ completionHandler: @escaping AttributionDetailsBlock) {
+    func requestAttributionDetails(_ completionHandler: @escaping AttributionDetailsBlock) {
```

Comment:
can we get rid of most external parameter names at this point too?

Reply:
Can you please clarify? I am not sure I understand what you mean by that

Reply:
i meant the "_", or any other extra parameter names, if they were added for objc compatibility (and on internal functions)

Reply:
Oh yeah, I think it might make sense to do a public API naming pass. Not necessarily for this PR.